components/Starknet/modules/architecture_and_concepts/nav.adoc

```diff
@@ -1,6 +1,6 @@
 * Architecture
 
-** Blockchain architecture
+** xref:Network_Architecture/introduction.adoc[Starknet Architecture]
```

Comment:
Logically, this makes sense, but a general UX principle is that it's not a good idea to have more than one function for a given UI element, In this case, I tried to separate out expandable/collapsible sections from content. The idea is that when you click on something, it either expands/collapses, OR it takes you to a page...but not both.
```suggestion
** Starknet architecture
*** xref:Network_Architecture/introduction.adoc[About Starknet architecture]
```

Reply:
This makes a lot of sense! Thanks @stoobie